### DISCUSSION TITLE
Generate/create reports of Projects, Tasks, Jobs from the database to pdf or CSV format

### DISCUSSION THREAD

**Discussion by keshavadk** [cvat-ai/cvat#4141] [Ideas]
How can we Generate/create reports of Projects, Tasks, Jobs from the database to pdf or CSV format in CVAT , which can be downloaded?

**Comment by nmanovic** [comment]
@keshavadk , it is not possible at the moment. Please submit a feature request.

**Reply by keshva23** [reply]
@nmanovic ok . Thank you.
